src/index.html

```diff
@@ -9,10 +9,74 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <ul class="calendar calendar--month-length-31 calendar--start-day-sun">
+        <li class="calendar__day"></li>
+
+        <li class="calendar__day"></li>
```

Comment:
You can remove the blank lines between the days
```suggestion
        <li class="calendar__day"></li>
        <li class="calendar__day"></li>
```